src/managers/AdjacencyManager.ts

```diff
@@ -0,0 +1,76 @@
+import { int, Nullable } from "../definitions/utils";
+import { AdjacencyMap } from "../utilities/AdjacencyMap";
+import tileConfigs from "../configs/tiles.json";
+import { PlaceTile } from "../definitions/placeTile";
+import { SessionDTO } from "../definitions/dto";
+import SessionManager from "./SessionManager";
+import { assert } from "../utilities/utils";
+import { api } from "../utilities/api";
+import TileManager from "./TileManager";
+import { pathRepresentation } from "../definitions/pathRepresentation";
+
+class AdjacencyManager {
+    private adjacencyMap: Nullable<AdjacencyMap>;
+    private pathMap: Map<int, pathRepresentation>;
+    private hasWon: boolean;
+
+    public constructor() {
+        this.adjacencyMap = null;
+        this.pathMap = this.createConnectionsMap();
+        this.hasWon = false;
+    }
+
+    public setHasWon(hasWon: boolean): void {
```

Comment:
Wouldn't it make more sense to put this method in the playermanager? just because it is player specific. What is your opinion about it?